addons/binding/org.openhab.binding.feed/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Feed Binding
+Bundle-SymbolicName: org.openhab.binding.feed;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jdom-2.0.2.jar,
+ lib/rome-1.5.1.jar,
+ lib/rome-utils-1.5.1.jar
+Import-Package: org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.openhab.binding.feed,
+ org.openhab.binding.feed.handler,
+ org.osgi.framework,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.feed,
+ org.openhab.binding.feed.handler,
+ com.rometools.rome.io,
```

Comment:
Why do you export the rome library? I assume this is required for the transformation service?
Imho, the transformation service should not depend on the presence of the binding, so you might rather want to move the lib to its own bundle (org.openhab.io.transport.feed?), which can then be used by the two add-ons. Ok for me to do this as a second step in the context of #937.